dbt/models/default/default.vw_pin_universe.sql

```diff
@@ -31,6 +31,7 @@ WITH pardat_adjusted_years AS (
 
 SELECT
     -- Main PIN-level attribute data from iasWorld
+    CONCAT(par.parid, par.taxyr) AS row_id,
```

Comment:
My plan was for this column to not even be visible on socrata. And since it won't exist long term in this view (generation will be done on the fly by the upload script) I'm not too worried about.